### DISCUSSION TITLE
why is it not possible to render soft lines with Graphics??

### DISCUSSION THREAD

**Discussion by ctwhome** [pixijs/pixijs#10252] [Q&A] [answered]
they get scattered, and if i remove the option to moveTo() on pointer move, all lines are rendered from the top left corner.
Code here: https://stackblitz.com/edit/stackblitz-starters-hkdedp?description=HTML/CSS/JS%20Starter&file=script.js,styles.css,index.html&terminalHeight=10&title=Static%20Starter
![image](https://github.com/pixijs/pixijs/assets/4195550/c9c6e1f3-1024-4593-a6e7-41ce5f4e96a8)

**Comment by bigtimebuddy** [comment, marked as answer]
Have you considered using RenderTexture for drawing instead? 
https://jsfiddle.net/bigtimebuddy/68gy2o4u/

Using Graphics to draw like this will have performance challenges as it get more complex.

**Reply by ctwhome** [reply]
thanks for the tip, i updated the code to use RenderTexture like in the example here https://stackblitz.com/edit/stackblitz-starters-hkdedp?file=index.html 
<img width="479" alt="image" src="https://github.com/pixijs/pixijs/assets/4195550/31a793c0-f9c3-4f68-89b0-ccb9ce1b0be2">
